src/styl/fonts/fonts.styl

```diff
@@ -0,0 +1,89 @@
+/* This stylesheet generated by Transfonter (https://transfonter.org) on August 30, 2017 7:40 PM */
+
+// Druk
+
+@font-face {
+	font-family: 'Druk';
+	src: url('../static/fonts/Druk-Bold.eot');
+	src: url('../static/fonts/Druk-Bold.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/Druk-Bold.woff') format('woff'),
```

Comment:
I think we should always check the box for woff2 on Transfonter (I don't understand why it isn't default) as woff2 files are about 1/3 of the size of woff files and widely supported